source/resources/DatasetResource.js

```diff
@@ -30,11 +30,16 @@ class DatasetResource {
     const badMapIds = []
     const badValueIds = []
     const parsed = csvParseRows(csv, d => [d[0], parseFloat(d[1])]).filter(row => {
-      const hasId = features.indexOf(row[0]) > -1
-      if (!hasId) {
-        badMapIds.push(row[0])
+      let hasId = (features.indexOf(row[0]) > -1)
+      if (!hasId && row[0]) {
```

Comment:
Rather than do the `indexOf` lookup a second time in a conditional, you might just sanitize all IDs on their way in:

```javascript
let id = row[0]
if (id.length < 2) { // pad with leading zero
  id = `0${id}`
}
```

…or something…